app/page/network/replication-in.js

```diff
@@ -0,0 +1,202 @@
+const { h, Value, Dict, onceTrue, computed, watch, watchAll, throttle } = require('mutant')
```

Comment:
wooah, nice! This file is really sweet. Gonna be testing this.

Reply:
you've already seen this in the network page right ... it's been in patchbay for a while!

or have you not seen `/network` !